modules/pipeline/pipengine/reconciler/listen.go

```diff
@@ -188,6 +188,8 @@ func makeContextForPipelineReconcile(pipelineID uint64) context.Context {
 	pCtx := context.WithValue(context.Background(), ctxKeyPipelineID, pipelineID)
 	exitCh := make(chan struct{})
 	pCtx = context.WithValue(pCtx, ctxKeyPipelineExitCh, exitCh)
+	executorCh := make(chan interface{})
+	pCtx = context.WithValue(pCtx, spec.CtxExecutorChKey, executorCh)
```

Comment:
executor is action level, not pipeline level.

Reply:
move the executor chan to traskRun framework